src/components/appmodallogin.main.less

```diff
@@ -57,10 +57,36 @@
 
   #login_modal_close_button {
     font-size: 30px;
+    text-align: right;
   }
 
   .form-input.btn-container {
     padding-top: 15px;
   }
 
+  .form-container {
+    margin: auto 30px;
+    text-align: center;
+  }
+
+  .form-control {
+    background: #f7f7f7 none repeat scroll 0 0;
+  }
+
+  .panel {
+    margin-bottom: 10%
+  }
+
+  @media (max-width: @mobileWidth - 1) {
```

Comment:
This feels a bit awkward around the 576-767px range. Have you taken a look at any of the additional options Bootstrap offers for positioning?
https://getbootstrap.com/docs/4.1/components/modal/